### PR TITLE
fix(statebar): add new case on stateBar

### PR DESCRIPTION
**Pull request by Lawndlwd**
## Type

- Feature

#### What is expected?

(Description of the new behavior)

when the stateBar value is between 70 && 90 the progress bar background color will be orange 

#### The following changes where made:

(Describe what you did)

1. add new condition on the variant when the value  is between 70 && 90  to display orange color

Capture d’écran 2022-03-04 à 17 33 47

### CODE REVIEW

File: src/components/StateBar/index.tsx [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { css } from '@emotion/react'
 import PropTypes from 'prop-types'
 import React, { ReactNode, useMemo } from 'react'
+import { colors } from '../../theme'
```
Comment (chambo-e) [inline src/components/StateBar/index.tsx:4]: We try to avoid importing directly from the theme file, could you use the injected Theme in emotion instead ?

**Review by chambo-e — CHANGES_REQUESTED** [review]
Small comment and also we shouldn't post internal links publicly, I edited your description :)

**Comment by matthprost** [comment]
Screenshot 2022-03-04 at 14 42 32

You can add an example in the documentation of storybook :)

**Comment by Lawndlwd** [comment]
> Screenshot 2022-03-04 at 14 42 32
> 
> You can add an example in the documentation of storybook :)

 Yes, and we have this issue that the imported color from theme wouldn't appear on storybook, any suggestion @matthprost

File: src/components/StateBar/index.tsx [hunk 2]
```diff
@@ -53,12 +53,15 @@ export const StateBarBar = ({
   value = 0,
   ...props
 }: StateBarBarProps): JSX.Element => {
+  const theme = useTheme()
+
   const variant = useMemo(() => {
     if (unlimited) return 'success'
```
Comment (philibea) [inline src/components/StateBar/index.tsx:59]: we can now using the new colors everywhere ? this will avoid misunderstood where the colors come from ?

Comment (philibea) [reply src/components/StateBar/index.tsx:59]:
```js  
if (unlimited) return 'success'
```

will be 
```js
  if (unlimited) return theme.colors.therightcolorstobeused ?
```

Comment (Lawndlwd) [reply src/components/StateBar/index.tsx:57]: Good point @philibea, there is an issue that I didn't understand `theme.colors.succes.background` or whatever color from the new colors don't work, don't work I mean the bar show without color on storybook like the photo above

Comment (chambo-e) [reply src/components/StateBar/index.tsx:57]: @philibea ProgressBar expect a variant, not a color :)

**Review by chambo-e — APPROVED** [review]

File: src/components/StateBar/index.tsx (hunk 2 above)
Comment (Lawndlwd) [reply src/components/StateBar/index.tsx:57]: @chambo-e  do you think that the history of variant need to be changed to accept color as well as variant?

Comment (chambo-e) [reply src/components/StateBar/index.tsx:57]: @Lawndlwd nope, a variant is a variant, if we start accepting colors we are going to be lost in no time 😄

**Review by philibea — APPROVED** [review]

**Pull request merged**